### DISCUSSION TITLE
Works in Cemu but not in Rpcs3

### DISCUSSION THREAD

**Discussion by alexandrage** [Berny23/LD-ToyPad-Emulator#66] [Q&A]
Works in Cemu but not in Rpcs3.

**Comment by Berny23** [comment]
Please check this additional guide for RPCS3: https://github.com/Berny23/LD-ToyPad-Emulator#rpcs3-cannot-detect-the-toy-pad

**Reply by Flamestarter1814** [reply]
I tried doing this but it did not work for me
the game still says that the toy pad is not connected and the website says that it can not detect the game
